homework 2 and 3

mkapry opened the pull request:
форма
файл+перетаскивание
геолокация

mkapry committed:
homework 2 and 3

mkapry committed:
Delete frount-track.iml

mkapry committed:
Delete misc.xml

mkapry committed:
Delete modules.xml

mkapry committed:
Delete vcs.xml

mkapry committed:
Delete workspace.xml

martinkomitsky commented:
В следующий раз, присылай несколько пулл реквестов:
* Ветка hw2 - PR 1 [hw2]
  * Находясь в ветке hw2 делаешь `git checkout -b hw3` и делаешь PR 2 [hw3]

martinkomitsky commented:
@mkapry а как собрать твой проект? У меня вебпак выдает ошибки
![image](https://user-images.githubusercontent.com/14847018/47961661-348c8600-e020-11e8-8425-98c1ab09eec1.png)

martinkomitsky reviewed (changes requested):
Исправить ошибки сборки, исправить слово `Frontend` в названии репозитория

mkapry commented:
![2018-11-04 11 59 49](https://user-images.githubusercontent.com/33937993/47962176-c947b180-e029-11e8-871b-835547638754.png)

mkapry commented:
даже не знаю что сказать, потому что у меня ошибок нет...

mkapry commented:
![2018-11-04 12 24 18](https://user-images.githubusercontent.com/33937993/47962361-c00c1400-e02c-11e8-996e-d45a3ce0140b.png)

martinkomitsky commented:
> не знаю что сказать

А кто знает? А.С. Пушкин?(((
прежде чем сдавать, проверяй, что у тебя начисто с нуля все заработает, как это делаю я:
- git clone твой репозиторий
- git checkout твоя ветка
- npm i
- npm run build 

Если все, как у тебя в исходном репозитории, то все супер и я буду проверять то же самое, что и есть у тебя.

А по моей ошибке можно легко найти, что в твоем package.json нет зависимости для лоадера.
https://github.com/webpack/webpack/issues/1470
Исправь, пожалуйста, и закоммить)

martinkomitsky commented:
и префикс `mkapry-` в названии излишен) сорри, пусть у всех одинаково будет

mkapry committed:
hw2and3

mkapry committed:
fixing smth
Merge branch 'hw2and3' of https://github.com/mkapry/mkapry-2018-FS-21-Frountend-Kaprielova into hw2and3

mkapry commented:
Собралось без ошибок, можно проверять!
Спасибо что объяснили как правильно делать)

martinkomitsky reviewed (approved):
![image](https://user-images.githubusercontent.com/14847018/48054046-4f483180-e1bd-11e8-8fd7-49a8081579fc.png)

Не получается получить текущую геолокацию. 
ДЗ-2: 5
ДЗ-3: 4

martinkomitsky commented:
>Спасибо

Пожалуйста! 

В следующий раз присылай отдельные PR:
>Ветка hw2 - PR 1 [hw2]
>Находясь в ветке hw2 делаешь git checkout -b hw3 и делаешь PR 2 [hw3]
